data/core/units/humans/Loyalist_Cavalryman.cfg

```diff
@@ -44,7 +44,6 @@ Cavalrymen are very useful for taking and holding positions on open ground, for
     [/movement_anim]
     {DEFENSE_ANIM "units/human-loyalists/cavalryman/cavalryman-defend2.png" "units/human-loyalists/cavalryman/cavalryman-defend1.png" {SOUND_LIST:HORSE_HIT} }
     [resistance]
-        blade=70
```

Comment:
I always get confused by resistances; remind me if this is a buff or a nerf?

Reply:
It is nerf. Mountes has blade=80 so it is 20% res now.